examples/hpc-slurm.yaml

```diff
@@ -88,6 +87,9 @@ deployment_groups:
     - h3_nodeset
     settings:
       partition_name: h3
+      # Note that H3 is available in only specific zones. https://cloud.google.com/compute/docs/regions-zones
+      # Choose another partition as the default if H3 is not available in your chosen zone.
```

Comment:
Move L90 above L78. Then don't bother with L91 comment, and also leave debug partition as default. The I would merge this.

Reply:
Ok. So, effectively this PR will add just the comment about zone availability 👍 .

Reply:
@nick-stroud Landed in git history mess. Since, this was a minor change and not much comment context to be retained, didn't think it was worth it to fix it. Created this [new PR](https://github.com/GoogleCloudPlatform/cluster-toolkit/pull/2854)